src/service/video-manager-impl.js

```diff
@@ -129,6 +129,63 @@ export class VideoManager {
       this.scrollListenerInstalled_ = true;
     }
   }
+
+  /**
+   * Returns whether the video is paused or playing after the user interacted
+   * with it or playing through autoplay
+   *
+   * @param {VideoEntry} entry
+   * @return {!../video-interface.VideoInterface} PlayingStates
+   * @private
+   */
+  getPlayingState_(entry) {
+    return entry.getPlayingState();
+  }
+
+  /**
+   * Returns whether the video was interacted with or not
+   *
+   * @param {VideoEntry} entry
+   * @return {boolean}
+   * @private
+   */
+  userInteracted_(entry) {
+    return entry.userInteracted();
+  }
+
+  /**
+   * Returns whether the video is paused or playing after the user interacted
+   * with it or playing through autoplay
+   *
+   * @param {!../video-interface.VideoInterface} video
+   * @return {!../video-interface.VideoInterface} PlayingStates
+   */
+  getPlayingState(video) {
+    for (let i = 0; i < this.entries_.length; i++) {
+      if (this.entries_[i].video === video) {
+        return this.entries_[i].getPlayingState();
+      }
+    }
+    return PlayingStates.PAUSED;
```

Comment:
we should throw if video is not found. Do `dev().assert(false, 'video element given to getPlayingState() does not exist')`.